### ISSUE TITLE
Contrastive Learning for Unpaired Image-to-Image Translation

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#36]
# TL;DR

This study applies contrastive learning, which is often used in unsupervised representation learning, to domain transformation by GAN. Based on the suggestion that each place in the image before and after the domain transformation should have a similar structure, they perform contrastive learning with the corresponding place as a positive sample and the other place as a negative sample.
![image](https://user-images.githubusercontent.com/24524018/89734129-20977300-da95-11ea-86fa-128304380953.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.15651

# Submission Dates(yyyy/mm/dd)
2020/07/30

# Authors and institutions
Taesung Park, Alexei A. Efros, Richard Zhang, Jun-Yan Zhu
- University of California, Berkeley 
- Adobe Research

# Methods

# Results

# Comments
ECCV2020
